src/item-manager.js

```diff
@@ -0,0 +1,25 @@
+class ItemManager {
+   constructor() {
+      this.items = [];
+   }
+
+   add(task) {
+      this.items.push(task);
+   }
+
+   remove(i) {
+      this.items.splice(i, 1);
+   }
+
+   clear() {
+      this.items.length = 0;
+   }
```

Comment:
I just tested it and I am actually very surprised it's working.
It's not a standard way of clearing array in javascript.
I suggested changing it to be in a more standard way like `this.items = []` or `this.items.splice(0, this.items.length)`. it's up to your choice

Reply:
Yes it surprised me too so decided to try it. 
Was recommended to use here:
https://stackoverflow.com/questions/1232040/how-do-i-empty-an-array-in-javascript/1234337#1234337

> If you need to keep the original array because you have other references to it that should be updated too, you can clear it without creating a new array by setting its length to zero:

Not particularly important in my use case so either of these methods will work:)